config/webpack/test.webpack.config.js

```diff
@@ -20,7 +20,13 @@ function getWebpackConfig(skyPagesConfig, argv) {
   const runCoverage = (!argv || argv.coverage !== false);
   skyPagesConfig.runtime.includeRouteModule = false;
   const ENV = process.env.ENV = process.env.NODE_ENV = 'test';
-  const srcPath = path.resolve(process.cwd(), 'src', 'app');
+
```

Comment:
This is probably more non-blocking feedback, but it stood out to me that we now have to check for `argv` twice.  What are your thoughts on slightly refactoring to something like the following (untested)?

```
  const argvCoverage = argv ? argv.coverage : '';
  const srcPathUnresolved = [process.cwd(), 'src', 'app'];
  const runCoverage = argvCoverage !== false;
  
  if (argvCoverage === 'library') {
      srcPathUnresolved.push('public');
  }

  const srcPath = path.resolve(...srcPathUnresolved);
  skyPagesConfig.runtime.includeRouteModule = false;
  const ENV = process.env.ENV = process.env.NODE_ENV = 'test';
```